Assignments/Program.cs

```diff
@@ -1,3 +1,52 @@
-﻿using System;
+﻿﻿// ---------------------------------------------------------------------------------------
+// Academy23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// A2.2 - Reverse Number Guessing Game (LSB to MSB)
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static System.ConsoleColor;
+namespace Academy;
+public class A2 {
+   public static void Main () => PerformGame ();
+
+   /// <summary>Performs the Reverse number guess game</summary>
+   public static void PerformGame () {
+      Random r = new ();
+      Console.WriteLine ("\x1B[4m" + "Reverse Number Guessing Game:-" + "\x1B[0m"
+         + "\r\nThink of a number between 1 and 100, and answer the following 7 questions so that I can guess your number");
+      Console.WriteLine ("Enter to start.");
+      Console.ReadKey ();
+      List<int> bitList = new () { 0, 0, 0, 0, 0, 0, 0 };
+      List<int> twoPowers = new () { 64, 32, 16, 8, 4, 2, 1 };
+      int n = 2;
+
+      //Question 1:
+      Console.WriteLine ();
+      Console.Write ("Is your number even or odd? (e/o): ");
+      string firstLSD = Console.ReadLine ();
+      if (firstLSD == "o" || firstLSD == "O") bitList[6] = 1;

```

Comment:
Instead of two check you can use to lower for comparison for the input variable.